ui/components/src/Legend/Legend.stories.tsx

```diff
@@ -45,13 +45,22 @@ function generateMockLegendData(count: number): LegendProps['data'] {
 // Simple wrapper to try to help visualize that the legend is positioned absolutely
 // inside a relative ancestor.
 const LegendWrapper = (props: LegendProps) => {
+  const {
+    options: { position },
+  } = props;
+
   return (
     <Box
       sx={{
-        position: 'relative',
-        width: props.width + 100,
-        height: props.height + 100,
         border: (theme) => `solid 1px ${theme.palette.divider}`,
+        position: 'relative',
+        width: position === 'Right' ? props.width + 100 : props.width,
```

Comment:
curious - why do we need to add 100 to the width here?

Reply:
Good callout. I should put a comment above here explaining what I'm doing. The Legend doesn't look very interesting by itself, especially when you're trying to look at how the right/bottom positioning works. I've put a little wrapper div with a border around it that's bigger than the legend to try to simulate the legend sitting next to a chart and make it easier to understand what the positioning looks like. I do this by adding some spacing to the width or height depending on the positioning.

I'll add a short comment summarizing this above the code for clarification.

Reply:
Done!